WalletWasabi.Fluent/Controls/PrivacyContentControl.axaml.cs

```diff
@@ -73,38 +94,5 @@ public bool ForceShow
 		set => SetValue(ForceShowProperty, value);
 	}
 
-	protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
-	{
-		if (Design.IsDesignMode)
-		{
-			return;
-		}
-
-		base.OnAttachedToVisualTree(e);
-
-		_disposable = new CompositeDisposable();
-
-		Services.UiConfig
-			.WhenAnyValue(x => x.PrivacyMode)
-			.Merge(this.WhenAnyValue(x => x.ForceShow))
-			.ObserveOn(RxApp.MainThreadScheduler)
-			.Subscribe(_ =>
-			{
-				var value = !Services.UiConfig.PrivacyMode || ForceShow;
-
-				IsPrivacyContentVisible = !value;
-				IsContentVisible = value;
-			})
-			.DisposeWith(_disposable);
-
-		PrivacyText = TextHelpers.GetPrivacyMask((int)NumberOfPrivacyChars);
-	}
-
-	protected override void OnDetachedFromVisualTree(VisualTreeAttachmentEventArgs e)
-	{
-		base.OnDetachedFromVisualTree(e);
-
-		_disposable?.Dispose();
-		_disposable = null;
-	}
+	private IObservable<bool> RevealContent { get; }
```

Comment:
Nullable warning fix

```suggestion
	private IObservable<bool> RevealContent { get; } = Observable.Empty<bool>();
```